fs/dafilesrv/dafilesrv.cpp

```diff
@@ -391,8 +391,12 @@ int main(int argc, const char* argv[])
     unsigned short  sslport;
     unsigned dedicatedRowServicePort = DEFAULT_ROWSERVICE_PORT;
 #ifdef _CONTAINERIZED
-    bool stream = strsame(config->queryProp("@application"), "stream");
-    connectMethod = stream ? SSLOnly : SSLNone;
+    // Use the "public" certificate issuer, unless it's visibility is "cluster" (meaning internal only)
+    const char *certScope = strsame("cluster", getComponentConfigSP()->queryProp("service/@visibility")) ? "local" : "public";
```

Comment:
very minor:  The following would be easier to read:

```
const char * visibility = getComponentConfigSP()->queryProp("service/@visibility");
const char *certScope = strsame("cluster", visibility) ? "local" : "public";
```

Reply:
agree, will change.